specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/analyticsItems_API.json

```diff
@@ -0,0 +1,401 @@
+                    "readOnly": true
+                },
+                "TimeModified": {
+                    "type": "string",
+                    "description": "Date and time in UTC of the last modification that was made to this item.",
+                    "readOnly": true
+                },
+                "Properties": {
+                    "properties": {
+                        "functionAlias": {
+                        "type": "string",
+                        "description": "A function alias, used when the type of the item is Function"
+                        }
+                    },
+                    "description": "A set of properties that can be defined in the context of a specific item type. Each type may have its own properties."
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Azure subscription ID."
+        },
+        "ResourceGroupNameParameter": {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group.",
+            "x-ms-parameter-location": "method"
+        },
+        "ResourceNameParameter": {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Application Insights component resource.",
+            "x-ms-parameter-location": "method"
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Client Api Version."
+        },
+        "ItemScopePathParameter": {
+            "name": "scopePath",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [
+                "analyticsItems",
+                "myanalyticsItems"
+            ],
+            "x-ms-enum": {
+                "name": "ItemScopePath",
+                "modelAsString": false
+            },
+            "description": "Enum indicating if this item definition is owned by a specific user or is shared between all users with access to the Application Insights component.",
+            "x-ms-parameter-location": "method"
+        },
+        "ItemScopeParameter": {
+            "name": "scope",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "shared",
+            "enum": [
+                "shared",
+                "user"
+            ],
+            "x-ms-enum": {
+                "name": "ItemScope",
+                "modelAsString": false
```

Comment:
also consider changing to `true`, see previous comment regarding breaking changes